.vscode/cspell.json

```diff
@@ -200,7 +200,7 @@
     "sdk/loadtesting/azure-developer-loadtesting/**",
     "sdk/clientcore/core/**",
     "sdk/clientcore/http-okhttp3/**",
-    "sdk/clientcore/annotation-processor/**",
+    "sdk/tools/annotation-processor/**",
```

Comment:
Just a nit on files like these, try to keep alphabetical order so things can be found more easily.